### DISCUSSION TITLE
Tech News Browser Extension

### DISCUSSION THREAD

**Discussion by LeeRenJie** [EddieHubCommunity/support#911] [General]
Just stumbled upon this FREE browser extension that updates tech news when you create a new tab! You can add tags to follow news that you are interested in!
![image](https://user-images.githubusercontent.com/67599130/107646239-b619f680-6cb4-11eb-9897-e64d2c92e77f.png)

Link here: https://api.daily.dev/get?r=LRJ

**Comment by eddiejaoude** [comment]
thanks for sharing 👍
